### PR TITLE
fix: create playground to evaluate self-contained styles

### PR DESCRIPTION
**Pull request by jbadan**
This pr leverages the existing playground to create a secondary one to be used to evaluate if styles are self contained and what would need to be done to have each component's scss file be self contained.

The playgrounds can be run at the same time in order to compare with `npm run start:playground` and `npm run start:playground:selfContained`. << That needs a new name 🤷‍♀ 

## Regular playground 
* `core.scss`
* `components/componentName.scss` 
* whichever pages are specified in the `css_deps` array, see: https://github.com/SAP/fundamental/blob/master/test/templates/action-bar/index.njk#L10
* It's worth noting here that we might want to reevaluate this so we can have a clear picture of how 
_all_ styles work together for consumers using `fiori-fundamentals.css` (aka `all.scss`)
>> Note: this has not changed from the original implementation

## Self contained playground
* uses only `components/componentName.scss`
* only includes components, no layouts, templates, content pages, etc

## Other notable changes: 
* added titles to playground page to help differentiate between the two
* added "Back to index" link
* moved styles from layout to `resources/layout.css` to prevent confusion with inline styles
* cleaned up layout for continuity
* removed unused files in `resources` directory

Screen Shot 2019-05-01 at 8 40 52 AM
Screen Shot 2019-05-01 at 8 59 30 AM
Screen Shot 2019-05-01 at 8 41 40 AM
Screen Shot 2019-05-01 at 8 41 47 AM

### CODE REVIEW

**Comment by jbadan** [comment]
Failing tests will be solved by https://github.com/SAP/fundamental/pull/1438

File: package.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
     "release:create": "create-release",
     "release": "./scripts/publish-release.sh",
     "start:playground": "node test/app.js",
+    "start:playground:selfContained": "PORT=3031 SELF_CONTAINED=true npm run start:playground",
```
Comment (greg-a-smith) [inline package.json]: Maybe `start:playground:scs` (where "scs" stands for "self-contained styles")?

File: test/app.js [hunk 2]
```diff
@@ -213,65 +217,28 @@ router.get('/:key', (req, res) => {
     } finally {
 
     }
-    signale.info(`Requested http://localhost:3030/${key}`);
-    res.render(`${key}/index`, Object.assign(GLOBALS, { id: key, component: getStarterData(), data, libs: getLibs(req.query.lib) }));
```
Comment (greg-a-smith) [inline test/app.js]: Is the `libs` no longer needed?

File: test/templates/index.njk [hunk 3]
```diff
@@ -10,9 +10,10 @@ li {
             font-size: 14px;
         }
 </style>
+<h1>{% if selfContained %}Self Contained Styles Playground{% else %}Playground{% endif %}</h1>
```
Comment (greg-a-smith) [inline test/templates/index.njk]: Minor nit: "Self Contained" should be "Self-Contained".

File: test/templates/layout.njk [hunk 4]
```diff
@@ -30,86 +26,43 @@
...
     </style>
-    <!-- {% if libs.ie %}<link href="/static/fiori-fundamentals-ie11.css" rel="stylesheet">{% endif %} -->
 </head>
 
-<body class="">
-    <header style="border-bottom: solid 6px lightgray; padding-bottom: 10px; margin-bottom: 10px;">
+<body>
+    <a href="../">Back to Index</a>
+    <header class="test-header">
+        <h1>{% if selfContained %}Self Contained Styles Playground{% else %}Playground{% endif %}: <code>{{ data.id }}</code></h1>
```
Comment (greg-a-smith) [inline test/templates/layout.njk]: Minor nit: "Self Contained" should be "Self-Contained".

File: test/app.js (hunk 2 above)
Comment (jbadan) [reply test/app.js]: It was never being used - it should have been living on a branch as it wasn't a fully developed feature.

File: package.json (hunk 1 above)
Comment (jbadan) [reply package.json]: so much better! thank you

File: test/visual-regression-tests/config/backstopConfig.js [hunk 5]
```diff
@@ -79,8 +78,6 @@ module.exports = {
       "--disable-dev-shm-usage"
     ]
   },
-  "asyncCaptureLimit": 1,
-  "asyncCompareLimit": 1,
```
Comment (jbadan) [inline test/visual-regression-tests/config/backstopConfig.js]: This is what was making the tests run so slowly

**Pull request merged**